gbpservice/neutron/config/nsf.ini

```diff
@@ -0,0 +1,13 @@
+[DEFAULT]
```

Comment:
Minimize config as in the case of multi_process_framework. Whatever can be derived from other config files such as neutron.conf should be removed from here.
Can all of this be derived from 'neutron.conf'